contracts/adapters/TributeNFT.sol

```diff
@@ -199,7 +205,9 @@ contract TributeNFTContract is AdapterGuard, IERC1155Receiver, IERC721Receiver {
         bytes calldata data
     ) external override returns (bytes4) {
         ProcessProposal memory ppS = abi.decode(data, (ProcessProposal));
-        require(ppS.dao.lockedAt() != block.number, "reentrancy guard");
+        ReimbursementData memory rData = ReimbursableLib.beforeExecution(
```

Comment:
I guess we could extract the contract to an new internal function and use the `reimbursable` modifier. The same needs to be done for the `onERC721Received` function